.isort.cfg

```
@@ -0,0 +1,14 @@
+# https://github.com/timothycrosley/isort/wiki/isort-Settings
+[settings]
+default_section = THIRDPARTY
+include_trailing_comma = true
+indent = 4
+known_frameworks_party = towncrier, sphinx
```

I believe the correct syntax is
```suggestion
known_frameworks = towncrier, sphinx
```

(https://pycqa.github.io/isort/#custom-sections-and-ordering)